src/flavors/mini-pwa.scss

```diff
@@ -457,10 +457,77 @@ th, td {
 @import '../mini/contextual';
 /*
   Custom style fixes and tweaks for contextual elements.
-*/
-.toast {
-  font-size: 0.875rem;
+  */
+  .toast {
+    font-size: 0.875rem;
+  }
+
+.modal {
+  position: fixed;
+  top: 0;
+  left: 0;
+  display: none;
+  width: 100vw;
+  height: 100vh;
+  background: rgba(0, 0, 0, 0.45);
+
+  .card {
+    margin: 0 auto;
+    max-height: 50vh;
+    width: 100vw;
+    overflow: auto;
+
+    .close {
+      position: relative;
+      top: 0.85rem;
+      left: calc(100% - 4.5rem);
+      padding: 10px;
```

Comment:
This must be converted into `rem` units if this makes it into the flavor. **This specific fix**, if needed, is something you should keep as-is, the module file will always set this to `0`.

`10px = 0.625rem`.